deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1112,12 +1112,7 @@ class ChatViewController: UITableViewController {
 
     func updateMessage(_ messageId: Int) {
         if messageIds.firstIndex(where: { $0 == messageId }) != nil {
-            markSeenMessage(id: messageId)
```

Comment:
`markSeenMessage` is not needed because `updateMessage` is called for outgoing messages only? or is it called implicitly somewhere else? why was `markSeenMessage` there at all?

Reply:
I think this part was really old. When we moved to the new Chat UI I've just c&p this from the messageKit version. markseen is completely wrong here I think. This if clause will be entered if the read receipt was received / the message state changes. markmessage needs to be implemented for newly received messages (among others), which is handled elsewhere.